.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,24 @@
+
+## What does this PR do?
+
+<!--
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+-->
+
+Fixes # (issue)
+
 # Before submitting
 
 - [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
 - [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
-- [ ] Did you make sure to update the docs?   
-- [ ] Did you write any new necessary tests?  
+- [ ] Did you make sure your PR does only one thing, instead of bundling different chnages together? Otherwise, we ask you create a separate PR for every change.
```

Comment:
```suggestion
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you create a separate PR for every change.
```